packages/react-hooks/src/useBalancesAll.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { DeriveBalancesAll } from '@polkadot/api-derive/types';
+import { useApi, useCall } from '@polkadot/react-hooks';
```

Comment:
Use direct imports fo the local ones rather, i.e. `{ useApi } from './ueApi';`

Reply:
Done